### DISCUSSION TITLE
First time user, weird error message with git clone

### DISCUSSION THREAD

**Discussion by p-magee** [community/community#39885] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

First time Git user and am stuck trying to clone an existing repository to which I have access.  When I use `git clone` + URL I am met what the following message `error: cannot run rpostback-askpass: No such file or directory`.  I am then prompted for a GitHub user name, which I enter and immediately get the same message.  Then I get prompted for GitHub password but am not able to type (the cursor is there, nothing happen when I strike a key).  Any ideas or resources would be hugely appreciated!

**Comment by airtower-luna** [comment]
That error message is odd, and not one I've seen. It might help if you mention your OS and how you installed Git, maybe people with a similar setup have an idea. If you want to dig in yourself, my first suspicion is a misconfigured [credential helper](https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage).

> Then I get prompted for GitHub password but am not able to type (the cursor is there, nothing happen when I strike a key).

That part is normal, Git will not echo anything as you type (not even stars), keep typing.

However typing your *password* won't work: You cannot use your password for `git clone` or other Git operations (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).
